pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -95,27 +94,27 @@ func (p *mockPolicy) Name() string {
 	return string(policyTypeMock)
 }
 
-func (p *mockPolicy) Start(s state.State) error {
+func (p *mockPolicy) Start(context.Context, state.State) error {
 	return p.err
 }
 
-func (p *mockPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Container) error {
+func (p *mockPolicy) Allocate(_ context.Context, s state.State, pod *v1.Pod, container *v1.Container) error {
```

Comment:
nit:
```suggestion
func (p *mockPolicy) Allocate(context.Context, s state.State, pod *v1.Pod, container *v1.Container) error {
```

and the same below.

Reply:
Ack, thanks. Updated!